test/specs/runApexTests.e2e.ts

```diff
@@ -35,12 +33,23 @@ describe('Run Apex Tests', async () => {
     // Create Apex class 3 and test
     await utilities.createApexClassWithTest('ExampleApexClass3');
 
-    // Push source to scratch org
+    // Push source to org
```

Comment:
This is a scratch org, so I think the comment on line 38/36 should remain with "scratch" in it.

Reply:
I'd leave it without "scratch" for consistency, and it's no wrong anyways, in the end it's an org
![image](https://github.com/forcedotcom/salesforcedx-vscode-automation-tests/assets/113132642/807f0531-ebe9-4d35-be46-0378795e2f72)

Reply:
OK